pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -115,7 +115,7 @@ func NewConfigObserver(
 				// static path at which we expect to find the encryption config secret
 				"/etc/kubernetes/static-pod-resources/secrets/encryption-config/encryption-config",
 			),
-			etcd.ObserveStorageURLs,
+			etcdendpoints.BackwardsCompatibleObserveStorageURLs,
```

Comment:
interesting namespaces don't include etcd, so you won't be triggered I don't think

Reply:
I also just realized my configmap lister setup isn't scoped to the etcd namespace

Reply:
Fixed